pkg/apis/authorization/types.go

```diff
@@ -38,6 +38,10 @@ type SubjectAccessReview struct {
 	Status SubjectAccessReviewStatus
 }
 
+// +genclient=true
+// +nonNamespaced=true
+// +noMethods=true
```

Comment:
> It means you have to manually create the Create methods in the generated_expansion.go file.

Nice. Does that not get overridden as soon as someone else regenerates? (not to get too off topic)